helm-charts/splunk-otel-collector/templates/NOTES.txt

```diff
@@ -79,3 +79,9 @@ Splunk Network Explorer is installed and configured.
 [WARNING] "clusterReceiver.eventsEnabled" parameter is deprecated. Soon it will be replaced with "clusterReceiver.k8sObjects".
           Upgrade guidelines: https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md#0670-to-0680
 {{ end }}
+{{- if eq .Values.logsEngine "" }}
+[WARNING] The helm charts is going to move from Fluentd to the native OpenTelemetry logs collection by default.
```

Comment:
I'd use `` instead of " when describing parameters.

Reply:
This is terminal output, not Markdown. We use "" in other warnings, so I wanted to keep it consistent